lib/main_containers.pm

```diff
@@ -87,18 +87,18 @@ sub load_host_tests_podman {
         load_container_engine_test('podman');
         load_image_test('podman');
         load_3rd_party_image_test('podman');
-        loadtest 'containers/podman_firewall';
+        loadtest 'containers/podman_firewall' unless is_openstack;
         loadtest 'containers/buildah' unless is_sle_micro;
         # https://github.com/containers/podman/issues/5732#issuecomment-610222293
-        loadtest 'containers/rootless_podman' unless (is_sle('=15-sp1'));
+        loadtest 'containers/rootless_podman' unless (is_sle('=15-sp1') || is_openstack);
     }
 }
 
 sub load_host_tests_docker {
     load_container_engine_test('docker');
     load_image_test('docker');
     load_3rd_party_image_test('docker');
-    loadtest 'containers/docker_firewall';
+    loadtest 'containers/docker_firewall' unless is_openstack;
```

Comment:
same question as for podman_firewall. What is the issue of running this module in OpenStack? Maybe a comment would help.